exercises/randomgenerator.js

```diff
@@ -1,20 +1,30 @@
 
 
 class RandomGenerator{
-	constructor(){ 
+	constructor(min, max){ 
 		//takes in the minimum and maximum values
+		this.min = min;
+		this.max = max;
+		if(!this.min && !this.max){
```

Comment:
this would only set defaults if BOTH were not defined.  Should do so on each individually

Also this would be a problem if it passed in 0.  It would treat it the same as undefined.